### PR TITLE
FRMW-83: Update Flexbox Mixins

### PR DESCRIPTION
**Pull request by bry4n**
JIRA: https://jira.rax.io/browse/FRMW-83

* ~~Removed~~ Added deprecation warning to top of `flexbox.less`
* Updated mixins to not depend on `flexbox.less`
* Re-ident modified files (commit: https://github.com/rackerlabs/encore-ui/commit/95f49b0a0364c14f983242920cba6699f2ed2a94)

### LGTMs
- [x] Dev LGTM
- [x] Dev LGTM

### Screenshots

Forms:

forms

Tabs

tabs

rxSelect

rxselect

Layout

layout

### CODE REVIEW

**Comment by CITguy** [comment]
Easier to view diff with [**?w=1**](https://github.com/rackerlabs/encore-ui/pull/1181/files?w=1)

**Comment by CITguy** [comment]
Please put `src/styles/flexbox.less` back. This file is bundled with the bower package and applications may rely on it. However, please add a large comment to the top of the file noting that it is deprecated and will be removed in a future version of the framework.

**Comment by bry4n** [comment]
@CITguy  Added `flexbox.less` back with deprecation message.

File: src/styles/flexbox.less [hunk 1]
```diff
@@ -1,3 +1,8 @@
+/*
+ * DEPRECATION: This file will be depercated and removed in a future version
```
Comment (CITguy) [inline src/styles/flexbox.less]: `will be depercated and removed` -> `is deprecated and will be removed`

File: src/styles/flexbox.less [hunk 2]
```diff
@@ -1,3 +1,8 @@
+/*
+ * DEPRECATION: This file is depercated and will be removed in a future version
```
Comment (CITguy) [inline src/styles/flexbox.less]: misspelled `deprecated`

Comment (bry4n) [inline src/styles/flexbox.less]: :+1:

**Comment by CITguy** [comment]
LGTM

**Comment by CITguy** [comment]
@bryan, please squash commits.

**Pull request merged**